SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslationManager/SpatialOSWorkerConnectionMock.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/Connection/SpatialOSWorkerInterface.h"
+
+#include "Interop/Connection/OutgoingMessages.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/SpatialLatencyTracer.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+// The SpatialOSWorkerConnectionMock is intended to be a very minimal implementation of a WorkerConnection which just records then swallows
+// any data sent through it. It is intended to be extended with methods to query for what data has been sent through it.
+//
+// Only a few methods have meaningful implementations. You are intended to extend the implementations whenever needed
+// for testing code which uses the WorkerConnection.
+
+class SpatialOSWorkerConnectionMock : public SpatialOSWorkerInterface
```

Comment:
Same comment as for `SpatialOSDispatcherMock`, looks more like a Spy to me.

Reply:
agreed, renaming.